lumen/ai/export.py

```diff
@@ -67,7 +68,7 @@ def format_output(msg: ChatMessage):
 def render_cells(messages: list[ChatMessage]) -> tuple[Any, list[str]]:
     cells, extensions = [], []
     for msg in messages:
-        if msg.user == 'Help':
+        if msg.user == 'Help' or (isinstance(msg.avatar, SVG) and msg.user == " "):
```

Comment:
Isn't checking `msg.user == " "` enough and then separately we should actually handle Image avatars?